### PR TITLE
✏️ Added role details to generated API spec

### PR DESCRIPTION
**Pull request by sreehari-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-1856
Issue Type: DOCUMENTATION

# Changes
Added role details to generated `/formio/roles` API spec 
Minimal refactoring to fix server crash

# How has the change been tested?

Manual verification, etc.

# Screenshots (if applicable)
image
image

### CODE REVIEW

**Review by abilpraju-aot — APPROVED** [review]
LGTM

File: forms-flow-api/src/formsflow_api/resources/formio.py [hunk 1]
```diff
@@ -26,6 +26,25 @@
 
 API = Namespace("Formio", description="formio")
 
+role = API.model(
```
Comment (sumesh-aot) [inline forms-flow-api/src/formsflow_api/resources/formio.py:29]: Yes, this is correct. We need to add this to all of the endpoints, not just roles endpoint.

**Comment by sreehari-aot** [comment]
@sumesh-aot 
`flake8` new version has incompatibilities with `flake8-quotes`
https://github.com/zheller/flake8-quotes/pull/111
This would be a fix from their side.
Should we specify the `flake8` previous version or wait till they merge?

**Comment by sreehari-aot** [comment]
Closing due to bad commit tree

**Pull request closed without merging**